docgen/src/Getting started.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Getting started
+title: Getting Started Guide
```

Comment:
Not sure about the Guide addition. This is really a getting started tutorial. I dont want to confuse users with the actual guide section. Is getting started alone not ok?